src/thorin/continuation.h

```diff
@@ -72,16 +76,18 @@ class App : public Def {
     }
 
     void jump(const Def* callee, Defs args, Debug dbg = {});
-    void verify() const;
+    bool verify() const;
 
     friend class World;
 };
 
 //------------------------------------------------------------------------------
 
 enum class CC : uint8_t {
-    C,          ///< C calling convention.
-    Device,     ///< Device calling convention. These are special functions only available on a particular device.
+    C,              ///< C calling convention.
+    Device,         ///< Device calling convention. These are special functions only available on a particular device.
+    DeviceHostCode, ///< Calling convention to denote continuations that are generated as device code.
+    Internal,       ///< External, but only for linking with artic or anyopt.
```

Comment:
why is there an "internal" CC ? if the continuation is internal it effectively doesn't matter, or does it ?